### ISSUE TITLE
[Feature Request]: Adding generation task to the queue

### ISSUE THREAD

**Issue by rollingcookies** [lllyasviel/stable-diffusion-webui-forge#659]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

When a task is sent for generation, it will be automatically added to the queue list, which is sequentially processed by the SD server. Such a function is well realized in Comfi.

### Proposed workflow

1. Ctrl+Enter > Adds the generation task to the queue
2. The [Clear Queue] button > will cancel the generation by the server and clear the queue list

That's all it takes!

**Comment by pvfreis** [comment]
As a workaround for not having this feature, I've been using this extension for a while now and it works great (also let's you save queued jobs and rerun them later) https://github.com/ArtVentureX/sd-webui-agent-scheduler

**Comment by rollingcookies** [comment]
> As a workaround for not having this feature natively, I've been using this extension for a while now and it works great (also let's you save queued jobs and rerun them later) https://github.com/ArtVentureX/sd-webui-agent-scheduler

Thank you! But this is another glitchy extension that doesn't work properly! It always fails if you use the hotkey to queue, and when you use the button, it works every once in a while. 

Here I am asking (very much) to make a simple function of queuing by Ctrl+Enter with the possibility to clear the queue. Without a bunch of unnecessary useless functionality that only adds bugs and clutters the interface.

**Comment by kyle215ps3** [comment]
> > As a workaround for not having this feature natively, I've been using this extension for a while now and it works great (also let's you save queued jobs and rerun them later) https://github.com/ArtVentureX/sd-webui-agent-scheduler
> 
> Thank you! But this is another glitchy extension that doesn't work properly! It always fails if you use the hotkey to queue, and when you use the button, it works every once in a while.
> 
> Here I am asking (very much) to make a simple function of queuing by Ctrl+Enter with the possibility to clear the queue. Without a bunch of unnecessary useless functionality that only adds bugs and clutters the interface.

I agree I had many problems with this and it's always really hard to edit the task as well if I missed something.

**Comment by Koitenshin** [comment]
I'm also asking for this, but with an added extra. "ComfyUI" and cmdr2 aka "EasyDiffusion" both have the queue feature, but only EasyDiffusion can pause the queue and resume it later without issues. It's extremely helpful, even if that interface is incredibly slow in comparison to the others.

**Comment by klayhamn** [comment]
i second this request - i like to try a lot of small tweaks and queuing makes it so much easier than having to wait for each run to finish before submitting my next twek

**Comment by lexmirnov** [comment]
That would be really great. Due to the nature of the technology it's crucial to have at least 10 versions of one prompt/setting to choose from, if we speak art creation.
